lib/libutee/include/pta_imx_manufacturing_protection.h

```diff
@@ -0,0 +1,35 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2018-2019. 2023 NXP
+ */
+#ifndef PTA_IMX_MP_H
+#define PTA_IMX_MP_H
```

Comment:
`PTA_IMX_MANUFACTURING_PROTECTION_H` folliwing the heade file name.

Reply:
Ok